docs/index.md

```diff
@@ -339,6 +341,24 @@ have a `kid` header with a defined key.
 
 Default is `False`.
 
+### JWT_TOKEN_ID
+
+Configure whether tokens have a `jti` token id claim (and refreshed tokens have a `orig_jti` claim).
+
+May be set to:
+
+* `off`: do not include token id claims in tokens
+* `include`: add token id claims to tokens, but continue accepting old tokens without them
+* `require`: add token id claims to tokens, and reject tokens that lack them
+
+The default has been to include these claims since version 1.17, so when upgrading
```

Comment:
writer fell asleep in the middle of the sentence? ;)

Reply:
Pretty much. Will fix when I get back to my desk later this morning.